docs/cli-help.txt

```diff
@@ -22,14 +22,11 @@ optional arguments:
   -v, --verbose         This flag can be used multiple times to incrementally
                         increase the verbosity of output, e.g. -vvv. (default:
                         -2)
-  -t, --text            Use the text output instead of the curses UI.
```

Comment:
Oh sorry. I missed that one. Yeah I suppose you should make a separate PR that we can merge during the release.